app/env.ts

```diff
@@ -30,6 +30,7 @@ export const schema = {
     COPPER_PROXY_ADDRESS: String,
     RPC_URL: String,
     YEARN_VAULTS_ENDPOINT: String,
+    OWLRACLE_API_KEY: String,
```

Comment:
@franzns can you make sure you add the api key to all deployment

Reply:
I can do that, if you can get it to me :)

Reply:
env var is configured everywhere